ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1803,13 +2132,98 @@
 				D0A226071A72E0E900D33B74 /* SignalSpec.swift */,
 				B696FB801A7640C00075236D /* TestError.swift */,
 				C79B64731CD38B2B003F2376 /* TestLogger.swift */,
+				530F8BB41D0DBC3200009C2C /* DynamicPropertySpec.swift */,
 			);
 			path = Swift;
 			sourceTree = "<group>";
 		};
 /* End PBXGroup section */
 
 /* Begin PBXHeadersBuildPhase section */
+		530F8B711D0DB70B00009C2C /* Headers */ = {
+			isa = PBXHeadersBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		53990BAA1D282481007C84EB /* Headers */ = {
+			isa = PBXHeadersBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				53990BAB1D282481007C84EB /* UISegmentedControl+RACSignalSupport.h in Headers */,
```

Comment:
Why are those header files added to the target? Since the target is a pure Swift one, those files should not be needed I think.